impl/src/main/java/org/ehcache/impl/internal/store/heap/holders/OnHeapValueHolder.java

```diff
@@ -23,7 +23,7 @@
 /**
  * @author Ludovic Orban
  */
-public abstract class OnHeapValueHolder<V> extends AbstractValueHolder<V> {
+public abstract class   OnHeapValueHolder<V> extends AbstractValueHolder<V> {
```

Comment:
I do not get that change, it is isolated in a commit but does not get rid of spaces, instead adds them. Can you remove the commit from that branch?